lib/docker-stdio.js

```diff
@@ -289,6 +332,42 @@ function createDockerFileReadStreamServer(opts, callback) {
 
         zlogin.stdout.pipe(socket);
     }
+
+    function onConnectionZoneNotRunning(socket) {
+        clearTimeout(serverTimeout);
+
+        socket.on('close', function () {
+            tcpServer.close();
+        });
+
+        if (norm.match(new RegExp('^/'))) {
```

Comment:
Is this always true from the [definition of norm](https://github.com/joyent/sdc-cn-agent/pull/9/files#diff-d0a55ce2bbbac6ac71591ec569f1a733R196)? If so, it might be better to assert() it.